hugolib/page.go

```diff
@@ -168,30 +168,55 @@ func (p *Page) Param(key interface{}) (interface{}, error) {
 	return p.Site.Params[keyStr], nil
 }
 
+// Author returns the first listed author for a page
 func (p *Page) Author() Author {
-	authors := p.Authors()
+	authors := p.getAuthors()
```

Comment:
Author and Authors should also be added to Node. We will eventually end up with one struct with front matter support for all page types, but in the first step we should just return the site authors for the Node version of these.